code/src/layouts/header/components/header-search-input-dropdown/HeaderSearchInputDropdown.scss

```diff
@@ -13,6 +13,11 @@
   height: 250px;
   overflow-y: auto;
 
+  &__skeleton {
+    height: 50px;
+    margin-inline: spacing(3);
```

Comment:
why `margin-inline` here ?

Reply:
Because I need just a horizontal margin. We can also make in this way: 
`maring: 0 spacing(3)`. But I prefer this variant